plugins/vsphere/rockcraft.yaml

```diff
@@ -0,0 +1,82 @@
+# Dockerfile: https://github.com/vmware-tanzu/velero/blob/v1.13.2/Dockerfile
+name: velero-plugin-for-vsphere
+summary: Velero Plugin for vSphere rock
+description: |
+    A rock containing the Velero Plugin for vSphere.
+
+    Velero gives you tools to back up and restore your Kubernetes cluster
+    resources and persistent volumes.
+license: Apache-2.0
+version: 1.5.2
+
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+  amd64:
+
+environment:
+  APP_VERSION: 1.5.2
+  LD_LIBRARY_PATH: /plugins
+
+# Services to be loaded by the Pebble entrypoint
+services:
+  install-plugin:
+    override: replace
+    startup: enabled
+    command: "bash /scripts/install.sh"
+    on-success: shutdown
+    on-failure: shutdown
+
+parts:
+  dependencies:
+    plugin: dump
+    source: https://gvddk-libs.s3-us-west-1.amazonaws.com/VMware-vix-disklib-7.0.2-17696664.x86_64.tar.gz
```

Comment:
I guess it makes sense to ship this along with the Rock instead.
Please add it to the repo along with a note on where we got this from.

Reply:
You mean adding the tar in this repo? Sure.

Reply:
Done.